runners/flink/src/test/java/org/apache/beam/runners/flink/translation/wrappers/streaming/io/UnboundedSourceWrapperTest.java

```diff
@@ -592,27 +591,55 @@ public void testSerialization() throws Exception {
       InstantiationUtil.serializeObject(flinkWrapper);
     }
 
+    @Test(timeout = 10_000)
+    public void testSourceDoesNotShutdown() throws Exception {}
```

Comment:
is this dead code, or does it do something I'm missing?

Reply:
No, that's a left-over from refactoring the tests.